azext_connect/custom.py

```diff
@@ -78,7 +78,7 @@ def connect(resource_group, aks = None, acr = None, webapp = None, sql = None,
     check_resource(service_list, resource_group)
     para_dict = parseParameter(para_list)
     deploy(service_list, resource_group, para_dict)
-    print('Service %s connected via %s.' %(' and '.join([s[1] for s in service_list]), connection_type))
+    print('Service %s connected via %s.' %(' and '.join([s[1] + ':' + para_dict[s[0]] for s in service_list]), connection_type))
```

Comment:
May we not use para_dict? Because the keys in para_dict for mysql and spring-cloud is not the name in service_list. The usage here implicitly require the equal key definitions for para_dict and service_list. We'd better add a services_name array for that.